airbyte-integrations/connectors/source-amazon-ads/metadata.yaml

```diff
@@ -33,6 +33,13 @@ data:
   releaseStage: generally_available
   releases:
     breakingChanges:
+      6.0.0:
+        message: "`sponsoredDisplayReportStream` stream have updated schemas."
```

Comment:
As a user, I would cope better with the issue (especially with the small delay we ask for this change) if we were to indicate that this is because of an API deprecation. Something like: "The Report API v2 is getting deprecated in favor of V3. Given this change, fields available through the `sponsoredDisplayReportStream` stream will change."

Reply:
agreed, updated